qase-playwright/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "playwright-qase-reporter",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
Reporter must have commons 2.0.9 to enable custom suites.

Reply:
No, these are not related things